reference/javascript.md

```diff
@@ -42,7 +42,7 @@ To use configuration values in JavaScript, use the [`pulumi.Config`] class. For
 
 ## TypeScript
 
-You can elect to write Pulumi programs in TypeScript to get additional verification and tooling benefits.  To use TypeScript, some additional steps are required, because you will be compiling code using the TypeScript compiler.
+You can elect to write Pulumi programs in TypeScript to get additional verification and tooling benefits. As of version 0.15.0, Pulumi supports TypeScript natively so you don't need to explicitly run `tsc` on your program before running `pulumi`
```

Comment:
Should we mention that you are free to continue doing so, if you have custom build steps, etc?

Reply:
FWIW - I suggested some specific text for this above in the PR.